engine.js

```diff
@@ -614,7 +614,12 @@ var resolveError = function(err) {
 
     /* hmda-macro */
     HMDAEngine.isValidNumLoans = function(hmdaFile) {
-        return true;
+        var respondentID = hmdaFile.transmittalSheet.respondentID;
+        var numLoans = hmdaFile.loanApplicationRegisters.length;
+        return apiGET('isValidNumLoans', [numLoans, respondentID])
```

Comment:
Given that suggested re-ordering of the params happened in the API, these will not match as written.